src/DssSpell.t.sol

```diff
@@ -101,48 +89,68 @@ contract DssSpellTest is DssSpellTestBase {
         }
     }
 
-    function testCollateralIntegrations() private { // make public to use
+    function testCollateralIntegrations() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // checkIlkIntegration(
-        //     "RETH-A",
-        //     GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
-        //     ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
-        //     addr.addr("PIP_RETH"),
-        //     true, /* _isOSM */
-        //     true, /* _checkLiquidations */
-        //     false /* _transferFee */
-        // );
+        checkIlkIntegration(
+            "RETH-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
+            addr.addr("PIP_RETH"),
+            true, /* _isOSM */
+            true, /* _checkLiquidations */
+            false /* _transferFee */
+        );
     }
 
-    function testIlkClipper() private { // make public to use
+    function testIlkClipper() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // // Insert new ilk clipper tests here
-        // checkIlkClipper(
-        //     "TOKEN-X",
-        //     GemJoinAbstract(addr.addr("MCD_JOIN_TOKEN_X")),
-        //     ClipAbstract(addr.addr("MCD_CLIP_TOKEN_X")),
-        //     addr.addr("MCD_CLIP_CALC_TOKEN_X"),
-        //     OsmAbstract(addr.addr("PIP_TOKEN")),
-        //     20_000 * WAD
-        // );
+        checkIlkClipper(
+            "RETH-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
+            addr.addr("MCD_CLIP_CALC_RETH_A"),
+            OsmAbstract(addr.addr("PIP_RETH")),
+            2400 * WAD
+        );
     }
 
-    function testNewChainlogValues() private { // make private to disable
+    function testNewChainlogValues() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // Insert new chainlog values tests here
-        // checkChainlogKey("STARKNET_TELEPORT_BRIDGE");
-        // checkChainlogKey("STARKNET_TELEPORT_FEE");
+        checkChainlogKey("RWA007_A_INPUT_CONDUIT");
+        checkChainlogKey("RWA007_A_JAR_INPUT_CONDUIT");
 
-        // checkChainlogVersion("1.14.4");
+        checkChainlogVersion("1.14.5");
+    }
+
+    function testRemoveChainlogValues() public { // make private to disable
```

Comment:
Can we move this test to its respective place so the tests order will be similar to spell-goerli?